dex/Cargo.toml

```diff
@@ -8,16 +8,16 @@ repository = "https://github.com/project-serum/serum-dex"
 edition = "2018"
 
 [features]
-program = ["solana-sdk/program", "spl-token/program", "spl-token/no-entrypoint"]
-client = ["solana-sdk/default", "spl-token/default"]
-fuzz = ["arbitrary", "solana-sdk/curve25519-dalek"]
-test = ["solana-sdk/default"]
-default = []
+program = []
+client = []
+fuzz = ["arbitrary"]
+test = []
+default = ["program"]
```

Comment:
Is it possible we opt-in "program"? 

It follows the original code's behavior.

Reply:
`cargo build-bpf` and the `solana-program` crate don't know about the "program" feature at all.  It's defined here because the Dex itself makes assumptions about the presence of this feature.  A more complete port would involve removing the usage of the "program" feature entirely from the Dex

Reply:
the CI error is 

```
error[E0425]: cannot find function `send_from_vault` in this scope
    --> dex/src/state.rs:1966:13
     |
1966 |             send_from_vault(
     |             ^^^^^^^^^^^^^^^ not found in this scope
error[E0425]: cannot find function `send_from_vault` in this scope
    --> dex/src/state.rs:1978:17
     |
1978 |                 send_from_vault(
     |                 ^^^^^^^^^^^^^^^ not found in this scope
error[E0425]: cannot find function `send_from_vault` in this scope
    --> dex/src/state.rs:2326:9
     |
2326 |         send_from_vault(
     |         ^^^^^^^^^^^^^^^ not found in this scope
error: aborting due to 3 previous errors
```

And `send_from_vault` is guarded with `program` feature, getting rid of it would help.

Reply:
okay. thanks for your info.